urbit/app/volt.hoon

```diff
@@ -1031,8 +1107,20 @@
           wach.chan
         (~(put by wach.chan) script-pubkey.funding-output id.new-channel)
       ==
+    =+  our-com=(rear our.commitments.new-channel)
```

Comment:
can use new get-balance scry if better

Reply:
This is for a single channel rather than total balance. Maybe we want to add a scry endpoint for getting an individual channel's balance?

Reply:
I'll use this on the front end to get total balance rather than summing channel balances

Reply:
Actually this is less convenient than summing channel balances because I would have to scry on a timer or something to keep the UI consistent, although I can still do that if you prefer. What the UI would want is a channel balance subscription, but that's probably hard to implement.